README.md

```diff
@@ -52,17 +58,17 @@ var err error
 // Create the default client
 client := pilosa.DefaultClient()
 
-// Create an Index object
-myindex, err := pilosa.NewIndex("myindex", nil)
+// Retrieve the schema
+schema, err := client.Schema()
 
-// Make sure the index exists on the server
-err = client.EnsureIndex(myindex)
+// Create an Index object
+myindex, err := schema.Index("myindex", nil)
 
 // Create a Frame object
 myframe, err := myindex.Frame("myframe", nil)
 
-// Make sure the frame exists on the server
-err = client.EnsureFrame(myframe)
+// make sure the index and frame exists on the server
+err = client.SyncSchema(schema)
 
 // Send a SetBit query. PilosaException is thrown if execution of the query fails.
```

Comment:
Minor comment - rather than "PilosaException is thrown", we should really be saying "PilosaError is returned" since Go doesn't have the notion of throwing exceptions.